bin/daemon

```diff
@@ -1,14 +1,76 @@
 #!/usr/bin/env ruby
 # Just keep her running
+require 'optparse'
+require 'ostruct'
+require 'digest/sha2'
+
+def calc_checksum
+  Digest::SHA2.file __FILE__
+end
+
+@checksum = calc_checksum
+
+def get_data
+  data = {}
+  data[:commit] = `git rev-parse HEAD`.chomp
+  data
+end
+
+def git_update
+  get_data
+end
+
+def make_command
```

Comment:
I honestly think it would be much easier if we did exec in `Scarlet.root`, `git rev-parse HEAD`, and just did this inside the command itself. Less parsing, less code, more simple.